### ISSUE TITLE
[Web] Accessibility feature behind a paid role - Dark mode

### ISSUE THREAD

**Issue by drummerXD** [mahloola/osuCollector-frontend#8]
From: https://blogs.perficient.com/2022/03/04/dark-mode-and-digital-accessibility-who-it-helps-and-without-it-who-it-may-hinder/
 "Some users have a disability that isn’t ongoing like others, so having the ability to easily toggle between dark mode and light mode without having to jump to settings or preferences makes it that much more user-friendly for all."

And from https://mobilea11y.com/quick-wins/dark-mode/#:~:text=Many%20people%20don't%20realise,also%20a%20valuable%20accessibility%20feature.
 "Many people don’t realise dark mode is an accessibility feature. It’s often just considered a nice to have, a cool extra feature that power users will love. But dark mode is also a valuable accessibility feature. Some types of visual impairment can make it painful to look at bright colours, or large blocks of white might wash over the black text. Some people with dyslexia or Irlen’s Syndrome can struggle to read black text on a white background. Like almost any accessibility feature, dark mode is about providing flexibility to your users so they can use your app in the way that is most comfortable to them."
  
My comments:
I shouldn't have to go into my windows and/or monitor settings to mess with my screen so I can comfortably look at the site for more than five minutes. Me personally, light mode can give me bad headaches, especially at night. Locking an accessibility feature behind a paid supporter role is the beyond one of the worst things you can do.

**Comment by FunOrange** [comment]
ok, i made it free
